macros/importChannelMap.py

```diff
@@ -0,0 +1,238 @@
+#! /usr/bin/env python
```

Comment:
Why is this in `macros` rather than `mapping`?
Is the only reason for creating a text file, rather than returning a mapping object, to maintain close consistency with the old v2 mapping?  Or is there a performance/operational reason?

Reply:
> * Easier to install

I don't follow this constraint

> * Easier to document

Nor this one

 > * No need to modify `PATH`
 > * I've never understood why `buildMapFiles.py` was put in a "module" directory in the first place

As a module you can simply do (without *any* modification of the `PATH`):
`python -m gempython.gemplotting.mapping.buildMapFiles`

Further comment in later reply...

Reply:
Move this to mapping.

Reply:
If the EDMS document is not accessible without a CERN login we will need to come up with another solution...some testing needed and expect a follow-up discussion.